docs/howto/write-logs.rst

```diff
@@ -205,3 +205,40 @@ To add custom configurations to ElasticSearch (e.g. turning on ``ssl_verify``, a
     use_ssl=True
     verify_certs=True
     ca_certs=/path/to/CA_certs
+
+
+.. _write-logs-stackdriver:
+
+Writing Logs to Stackdriver
+------------------------------
+
+Airflow can be configured to read and write task logs in `Google Stackdriver Logging <https://cloud.google.com/logging/>`__.
+
+Follow the steps below to enable Stackdriver logging:
+
+To enable this feature, ``airflow.cfg`` must be configured as in this
+example:
+
+.. code-block:: ini
+
+    [core]
+    # Airflow can store logs remotely in AWS S3, Google Cloud Storage or Elastic Search.
+    # Users must supply an Airflow connection id that provides access to the storage
+    # location. If remote_logging is set to true, see UPDATING.md for additional
+    # configuration requirements.
+    remote_logging = True
+    remote_base_log_folder = stackdriver://logs-name
+    remote_log_conn_id = custom-conn-id
+
+All configuration options are in the ``[core]`` section.
+
+The value of field ``remote_logging`` must always be set to ``True`` for this feature to work.
+Turning this option off will result in data not being sent to Stackdriver.
+The ``remote_base_log_folder`` option option contains the URL that specifies the type of handler to be used.
+For integration with Stackdriver, this option should start with ``stackdriver:///``.
+The path section of the URL specifies the name of the log e.g. ``stackdriver://airflow-tasks`` writes
+logs under the name ``airflow-tasks``.
+
+By using the ``logging_config_class`` option you can get advanced features of this handler.
```

Comment:
It's not clear how to replace logging_config_class using remote_base_log_folder. It should be possible to specify the  class via ? parameter I think.

Reply:
I updated the description. Now we have section about ``logging_config_class``.